src/components/pill-selector-dropdown/AvatarPill.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import noop from 'lodash/noop';
+import classNames from 'classnames';
+import X from '../../icon/fill/X16';
+// $FlowFixMe this imports from a typescript file
+import LabelPill from '../label-pill';
+import Avatar from '../avatar';
+
+import './AvatarPill.scss';
+
+type Props = {
+    avatarUrl?: string,
+    className?: string,
+    hasWarning?: boolean,
+    id?: string | number,
```

Comment:
we should consolidate around `string | number | null`

Reply:
adding null is an incompatible type so keeping it as is